docs/operator/operations.md

```diff
@@ -0,0 +1,100 @@
+# Lifecycle Manager - Operations
+
+## Metrics
+
+Lifecycle Manager metrics are exposed over port `8080` through the metrics endpoint `/metrics`.
+
+The following metrics are exposed:
+
+| Metric Name                              | Metric Type    | Description                                                                                  |
+|------------------------------------------|----------------|----------------------------------------------------------------------------------------------|
+| `lifecycle_mgr_requeue_reason_total`     | Counter Vector | Indicates the requeue reason of a reconciler.                                                |
+| `lifecycle_mgr_kyma_state`               | Gauge Vector   | Indicates the state of a Kyma CR.                                                            |
+| `lifecycle_mgr_module_state`             | Gauge Vector   | Indicates the state of a module added to a Kyma CR.                                        |
+| `lifecycle_mgr_mandatory_modules`        | Gauge          | Indicates the number of mandatory ModuleTemplate CRs.                                           |
+| `lifecycle_mgr_mandatory_module_state`   | Gauge Vector   | Indicates the state of a mandatory module added to a Kyma CR.                              |
+| `reconcile_duration_seconds`             | Gauge Vector   | Indicates the duration of a Manifest CR reconciliation in seconds.                              |
+| `lifecycle_mgr_purgectrl_time`           | Gauge          | Indicates the average duration of purge reconciliation.                                      |
```

Comment:
Where can we find the definition for "purge reconciliation"? I do have an idea, however the target audience is people who do not know about kyma deeply, and the goal is to give them knowledge.